src/main/java/org/NewTicTacToe/Main.java

```diff
@@ -1,9 +1,51 @@
+
 package org.NewTicTacToe;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+/**
+
+ Main class to run the Tic Tac Toe game
+
+ */
 public class Main {
+    /**
+
+     Main class to run the Tic Tac Toe game
+
+     */
     public static void main(String[] args) {
-        System.out.println("Hello world!");
-        System.out.println("Hello world!");
-        System.out.println("Hello world!");
+        Player player1 = null; // Initialize player1 as null
+        Player player2 = null; // Initialize player2 as null
+        Board board = null; // Initialize board as null
+        Scanner userInput = new Scanner(System.in); // Scanner to get input from user
+        System.out.println("Do you want to continue a saved game? (yes/no)");
+        String response = userInput.nextLine(); // Get response from user
+
+        StateManager stateManager = new StateManager(); // Create object of StateManager
+
+        if (response.equals("yes")) {
+            board = stateManager.loadGame(); // Load the saved game
```

Comment:
Maybe it would be nice to add a new if condition to check if an old game exists. Maybe the user says yes but the game does not exist.